src/device-registry/app.js

```diff
@@ -49,14 +57,17 @@ const runKafkaConsumer = async () => {
       },
     });
   } catch (error) {
-    logElement("KAFKA CONSUMER RUN ERROR", error.message);
+    logger.error("KAFKA: internal server error", error.message);
   }
 };
 
-runKafkaConsumer();
+if (constants.ENVIRONMENT === "STAGING ENVIRONMENT") {
```

Comment:
R KAFKA consumer in STAGING environment ONLY, for now...... @NoahNsimbe / @123MwanjeMike